examples/cube.py

```diff
@@ -1,5 +1,7 @@
 """
 This example renders a simple textured rotating cube.
+
+# test example: true
```

Comment:
The comment is in the docstring, is this intended?

Nitpick, but `test_example` (with an underscore) feels more natural because it reads a bit like a variable.

Reply:
Not intended. I just need something and I don't really care what it looks like, so in this case I'll just defer to your judgement. :)